themes/origins/assets/component-slider.js

```diff
@@ -0,0 +1,216 @@
+class Slider extends HTMLElement {
+  static observedAttributes = ["autoplay", "interval", "responsive"];
```

Comment:
Maybe we can make the slider responsive by default? This way we don't have to add that attr every time we use yc-slider